app/index.html

```diff
@@ -338,16 +338,18 @@ <h2 class="underline">Fully interoperable</h2>
   &lt;dom-module id=&quot;contact-card&quot;&gt;
     &lt;style&gt;...&lt;/style&gt;
     &lt;template&gt;
-      &lt;content&gt;&lt;/content&gt;
+      &lt;slot&gt;&lt;/slot&gt;
       &lt;iron-icon icon=&quot;star&quot; hidden$=&quot;{{!starred}}&quot;&gt;&lt;/iron-icon&gt;
     &lt;/template&gt;
     &lt;script&gt;
-      Polymer({
-        is: 'contact-card',
-        properties: {
-          starred: Boolean
+      class ContactCard extends Polymer.Element {
+        static get is() { return "contact-card"; }
+          constructor() {
```

Comment:
Oh sorry, I think the `static` and `constructor` should also left align (maybe the easiest is to indent `static` by 2?)